include/woff2/decode.h

```diff
@@ -19,7 +19,6 @@
 
 #include <stddef.h>
 #include <inttypes.h>
-#include "./woff2_out.h"
```

Comment:
I just realized that woff2_out should be a public header too. Sorry I've been confused by a comment by @khaledhosny on the old PR, regarding port.h ; I'll update the PR later.